### PR TITLE
ENH: update for AquaCrop output in LIS

### PR DESCRIPTION
**Pull request by gdelannoy**
Minor updates to reflect the reading of AquaCrop output generated by LIS, and to document choices made in various publications.

### CODE REVIEW

File: subroutines/read_LIS_output.m [hunk 1]
```diff
@@ -105,11 +112,23 @@
...
+        LIS_veg_swe(t_ind,:)=S(domain);
+      else
+        S=ncread(fname,'SoilMoist_tavg_1');
+	S1=S(domain);
+	S=ncread(fname,'SoilMoist_tavg_2');
+        S2=S(domain);
+        S=ncread(fname,'SoilMoist_tavg_3');
+        S3=S(domain);
+        LIS_ssm_sd(t_ind,:)=S1*weights(1) + S2*weights(2) + S3*weights(3);
+        S=ncread(fname,'WCMV1V2_tavg');
```
Comment (lbusschaert) [inline subroutines/read_LIS_output.m:139]: Out of curiosity: is that the Badj?

**Review by lbusschaert — APPROVED** [review]
Looks good, just one comment for my own knowledge.
Also: what's the difference between read_LIS_output.m and read_LIS_output_par.m?

File: subroutines/read_LIS_output.m (hunk 1 above)
Comment (gdelannoy) [reply subroutines/read_LIS_output.m:139]: Indeed; chances are that this variable name will change when we set up official AquaCrop-in-LIS output.

**Comment by gdelannoy** [comment]
> Looks good, just one comment for my own knowledge. Also: what's the difference between read_LIS_output.m and read_LIS_output_par.m?

Good point, I now removed the parallellized version of the LIS-reader to avoid redundancy.

**Review by mbechtold — APPROVED** [review]
Great, looks good to me as well!

**Pull request merged**